css/components/page/home/_home.scss

```diff
@@ -0,0 +1,98 @@
+.l-home-header {
```

Comment:
Clearly this is a layout, not a component. Move this file to layouts folder, please.